cookbooks/aws-parallelcluster-install/resources/ephemeral_drivers/ephemeral_drives_ubuntu20.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+provides :ephemeral_drives, platform: 'ubuntu', platform_version: '20.04'
+unified_mode true
+default_action :install
+
+use 'partial/_ephemeral_drives_common.rb'
+
+action_class do
+  def network_target
+    'network.target'
```

Comment:
This appears to be the default value. I wonder if we can have `ephemeral_drives.rb` with default value and `ephemeral_drives_redhat8.rb` override.